stdlib/crypto/src/main/ballerina/crypto/natives.bal

```diff
@@ -26,6 +26,18 @@ documentation { The `SHA256` hashing algorithm }
 documentation { The `MD5` hashing algorithm }
 @final public Algorithm MD5 = "MD5";
 
+documentation {
+    The encoding types supported for the HMAC key, by this package.
+}
+public type Encoding "UTF-8"|"BASE64"|"HEX";
+
+documentation { The `UTF-8` encoding" }
```

Comment:
There is an additional `"` towards the end of the doc data. Please correct the same mistake for the `SHA-1` algo with your PR. You can see it in the doc page for crypto package https://ballerina.io/learn/api-docs/ballerina/crypto.html.